packages/tradershub/src/features/cfd/modals/CTraderSuccessModal/CTraderSuccessModal.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useCallback } from 'react';
+import { useHistory } from 'react-router-dom';
+import { useBreakpoint, Button } from '@deriv/quill-design';
+import { ButtonGroup, Modal } from '../../../../components';
+import { Provider } from '@deriv/library';
+import { PlatformDetails } from '../../constants';
+import { CFDSuccess } from '../../screens';
+
+type TCTraderSuccessModal = {
+    isDemo: boolean;
+};
+
+const CTraderSuccessModal = ({ isDemo }: TCTraderSuccessModal) => {
+    const { isMobile } = useBreakpoint();
+    const history = useHistory();
+    const { hide } = Provider.useModal();
+
+    const renderButtons = useCallback(
+        () =>
+            isDemo ? (
+                <ButtonGroup>
+                    <Button className='rounded-200 ' onClick={() => hide()} size='lg'>
+                        Continue
+                    </Button>
+                </ButtonGroup>
```

Comment:
Isit necessary to wrap it inside of <ButtonGroup/> ? We are rendering just one button. If so I think you can place the if statement within the <ButtonGroup></ButtonGroup>. Looks cleaner and less repetitive